backend/src/aggregator.rs

```diff
@@ -131,6 +132,8 @@ pub struct AddNode {
     pub conn_id: ConnId,
     /// Recipient for the initialization message
     pub rec: Recipient<Initialize>,
```

Comment:
```suggestion
    pub rec: Addr<NodeConnector>,
```

And remove the `mute` recipient, no need to have two handles to the same actor.